connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -349,6 +350,7 @@ export async function syncNonThreaded(
   connectorId: ModelId,
   isBatchSync = false
 ) {
+  Context.current().heartbeat();
```

Comment:
don't think this serves any purpose since there is no heartbeat timeout

Reply:
since i've added sleep, we may have more ? i don't think it hurts to add heartbeat in running process ?

Reply:
It is useless unless there's a heartbeat timeout AFAIK

Reply:
removed heartbeats